src/main/java/com/android/volley/toolbox/HttpResponse.java

```diff
@@ -70,11 +89,28 @@ public final int getContentLength() {
         return mContentLength;
     }
 
+    /**
+     * If a byte[] was already provided by an HTTP stack that natively supports returning one, this
+     * method will return that byte[] as an optimization over copying the bytes from an input
+     * stream. It may return null, even if the response has content, as long as mContent is
```

Comment:
Make method @Nullable. Same for #getContent. I realize that one already could be null, so it may have just been an oversight previously